models.go

```diff
@@ -793,3 +811,85 @@ type FeatureEncodingOptions struct {
 	// instead of arrays in the response.
 	EncodeStructsAsObjects bool `json:"encode_structs_as_objects"`
 }
+
+type QueryContextValue interface {
+	isQueryContextValue()
+	ToProto() (*structpb.Value, error)
+}
+
+type StringValue string
+type FloatValue float64
+type BoolValue bool
+
+func (StringValue) isQueryContextValue() {}
+func (FloatValue) isQueryContextValue()  {}
+func (BoolValue) isQueryContextValue()   {}
+
+type QueryContext map[string]QueryContextValue
```

Comment:
For context, this is intentionally an entity unrelated to the `OnlineQueryContext` proto?

Reply:
this is what would go in the http request that corresponds to the proto